src/test/java/api/loans/scenarios/CheckInAgedToLostItemTest.java

```diff
@@ -17,4 +24,29 @@ protected void performActionThatRequiresRefund(AgeToLostFixture.AgeToLostResult
       .at(servicePointsFixture.cd1())
       .forItem(result.getItem()));
   }
+
+  @Test
+  public void canRefundFeesForLostAndPaidItemsThatWasAgedToLost() {
```

Comment:
As this is something the system should automatically do, I think `should` fits better than `can` what do you think?

```suggestion
  public void shouldRefundFeesForLostAndPaidItemsThatWasAgedToLost() {
```